config/locales/views/en.yml

```diff
@@ -632,6 +632,14 @@ en:
         link_tos: Terms of Service
         welcome_text: Hi! It looks like you've just logged into the Archive for the first time. %{help_link} or dismiss this message permanently.
   works:
+    adult:
+      caution: This work could have adult content. If you continue you have agreed that you are willing to see such content.
```

Comment:
**Nitpick:** I think there should be a comma after the initial clause in "if you continue..."
```suggestion
      caution: This work could have adult content. If you continue, you have agreed that you are willing to see such content.
```

Reply:
I fully agree and actually mentioned that when getting approval... no one explicitly agreed, though, so I intentionally left it as-is. I guess we can do it and then revert if anyone complains >.>